src/background/adblocker.js

```diff
@@ -86,6 +86,7 @@ export async function reloadMainEngine() {
     await engines.create(engines.MAIN_ENGINE);
     console.info('[adblocker] Main engine reloaded with no filters');
   }
+  contentScripts.clear();
```

Comment:
This is unrelated to reloading main engine, or if it is, it should be commented - for now it looks like leftover code.

Reply:
this is very much related to reloading of the main engine. When filters change we have to clear the content script so the new version can be applied

Reply:
At least add `__PLATFORM__` check, so `contentScripts` can be tree-shaked.